### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by kevinkouketsu** [SOCI/soci#1113] [General]
Is there a set deadline for the launch of a new release?

I am currently using the project through Conan, and unfortunately, it still contains the enum bug for PostgreSQL(same for vcpkg). It can't parse user-defined enum columns. 

It has been almost two years since the last release. Of course, I can use my recipe, have my fork from conan.io, etc, but would be nice to have an official version.

Are there any plans for an upcoming one?

**Comment by vadz** [comment]
There is currently one big [open issue](https://github.com/SOCI/soci/milestones/4.1.0) that should really be addressed before the next release.

**Reply by Spixmaster** [reply]
All related issues to the milestone for v4.1.0 are closed. What is still missing for the new release?

**Comment by kevinkouketsu** [comment]
Any news on this?

**Comment by marlowa** [comment]
As of 20270731 there are no official releases, just tags. Can we have official releases please? An official release should employ semantic versioning and have release notes and change log.

**Reply by vadz** [reply]
There are official releases, they are simply not on GitHub.

**Reply by marlowa** [reply]
I stand corrected. I thought github was the place to go to track the evolution of soci. Is it still being done on sourceforge then? If so, then I will go there.
